src/runtime/disco/ccl/utils.h

```diff
@@ -19,22 +19,21 @@
 #ifndef TVM_RUNTIME_DISCO_NCCL_UTILS_H_
 #define TVM_RUNTIME_DISCO_NCCL_UTILS_H_
 
-#include <nccl.h>
 #include <tvm/runtime/data_type.h>
 #include <tvm/runtime/disco/session.h>
 
 #include "../utils.h"
 
 namespace tvm {
 namespace runtime {
-namespace nccl {
+namespace ccl {
 
-#define NCCL_CALL(cmd)                                       \
-  do {                                                       \
-    ncclResult_t r = cmd;                                    \
-    if (r != ncclSuccess) {                                  \
-      LOG(FATAL) << "NCCLErrror: " << ncclGetErrorString(r); \
-    }                                                        \
+#define NCCL_CALL(cmd)                                             \
```

Comment:
```suggestion
#define CCL_CALL(cmd)                                             \
```
Do you think it is necessary to do a renaming of functions in `utils.h`?